src/webgpu/shader/execution/expression/call/builtin/acosh.spec.ts

```diff
@@ -31,7 +36,14 @@ g.test('f32')
   .params(u =>
     u.combine('inputSource', allInputSources).combine('vectorize', [undefined, 2, 3, 4] as const)
   )
-  .unimplemented();
+  .fn(async t => {
+    const makeCase = (n: number): Case => {
+      return makeUnaryF32IntervalCase(n, ...acoshIntervals);
```

Comment:
Is having `...acoshIntervals` better than `acoshAlternativeInterval, acoshPrimaryInterval`?
Explicitly stating the functions is how other tests currently handle this, but I considering converting them over to having an array of functions like this, so the tests don't have to iterate all of the options.

Reply:
This looks fine. I'd say coders choice, but this is slightly more preferable, IMHO.